server/src/main/java/com/flipkart/varadhi/web/v1/produce/ProduceHandlers.java

```diff
@@ -71,7 +71,8 @@ public List<RouteDefinition> get() {
     public void produce(RoutingContext ctx) {
         //TODO:: Request Validations pending
 
-        byte[] payload = ctx.body().asPojo(byte[].class);
+        byte[] payload = ctx.body().buffer().getBytes();
```

Comment:
in other PR, you mentioned it creates a copy. skippable?

Reply:
May be we can just use ctx.body().buffer().getByteBuf()?

Reply:
As of now continuing with ctx.body().buffer().getBytes() (which involves extra copy). Added a TODO to look this into independent of current PR.
getByteBuf() gives access to backing array, but would require Varadhi to maintain (or consume based on) start/end offset of the messages, need more thoughts.